### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by zihejia**
Merge 4.0.0.beta branch to master to release 4.0.0

### CODE REVIEW

File: Mixpanel-swift.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'Mixpanel-swift'
-  s.version = '2.10.4'
+  s.version = '3.0.0.beta.6'
```
Comment (jaredmixpanel) [inline Mixpanel-swift.podspec:3]: keep the beta tag?

File: Mixpanel.xcodeproj/xcshareddata/xcschemes/Mixpanel.xcscheme [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1200"
+   LastUpgradeVersion = "1310"
```
Comment (jaredmixpanel) [inline Mixpanel.xcodeproj/xcshareddata/xcschemes/Mixpanel.xcscheme:3]: all the schemes have this change, did we want this?

File: Mixpanel-swift.podspec (hunk 1 above)
Comment (zihejia) [reply Mixpanel-swift.podspec:3]: yeah, keep it until merged.

**Review by jaredmixpanel — APPROVED** [review]
lgtm

File: Mixpanel.xcodeproj/xcshareddata/xcschemes/Mixpanel.xcscheme (hunk 2 above)
Comment (zihejia) [reply Mixpanel.xcodeproj/xcshareddata/xcschemes/Mixpanel.xcscheme:3]: this is fine

**Pull request merged**